apps/target_apps_store.py

```diff
@@ -70,26 +70,55 @@ def images_size(self, target: FactoryClient.Target):
 
         return app_image_size
 
-    def copy(self, target: FactoryClient.Target, dst_dir: str):
+    def copy(self, target: FactoryClient.Target, dst_images_dir: str, dst_apps_dir: str):
+        if not self.exist(target):
```

Comment:
this is for backwards compatibility, correct? This conditional branch can eventually be removed?

Reply:
yes